gladier/client.py

```diff
@@ -432,15 +453,15 @@ def check_input(self, tool, flow_input):
         exception if the check does not 'pass'
 
         :param tool: The gladier.defaults.GladierDefaults tool set in self.tools
-        :param flow_input: Flow input intended to be passed to start_flow()
+        :param flow_input: Flow input intended to be passed to run_flow()
         :raises gladier.exc.ConfigException
         """
         for req_input in tool.required_input:
             if req_input not in flow_input['input']:
                 raise gladier.exc.ConfigException(
                     f'{tool} requires flow input value: "{req_input}"')
 
-    def start_flow(self, flow_input=None, use_defaults=True):
+    def run_flow(self, flow_input=None, use_defaults=True):
```

Comment:
This is a good change, thanks for including it!

Reply:
Yeap. Just fastforwarding the changes we did during the talk.